### DISCUSSION TITLE
Disabling right sidebar

### DISCUSSION THREAD

**Discussion by burntPlant** [google/docsy#1881] [Q&A]
Hello together,

is it somehow possible to disable the sidebar on the right handside completely? I couldn't find anything in the docsy documentation about it. I have circled the sidebar I mean in the picture. I would like to disable it globally.

Thanks!
![Screenshot (191)](https://github.com/google/docsy/assets/125071829/787cac54-ff1c-457a-b72a-f0561515febc)

**Comment by fekete-robert** [comment]
Technically you can hide the sidebar from your project's css like this:
`.td-sidebar-toc {
display: none;
}`

But that would just hide its contents, leaving empty white space behind. So it won't resize the main content section to be full-width (that would require customizing the classes of the main tag in the template files).
